themes/cypress/languages/ru.yml

```diff
@@ -0,0 +1,217 @@
+menu:
+  guides: Руководства
+  api: API
+  plugins: Плагины
+  faq: ЧаВо
+  examples: Примеры
+  search: Поиск
+
+index:
+  get-started: Быстрый старт
+
+page:
+  contents: Оглавление
+  back-to-top: Наверх
+  improve: Улучшить этот документ
+  prev: Назад
+  next: Вперёд
+  last-updated: "Последнее обновление: %s"
```

Comment:
Hmm, why "%s"?

Reply:
I don't know, copied from here https://github.com/cypress-io/cypress-documentation/blob/b8a035ca4b4d1e9deaa7721216865330d7cccd96/themes/cypress/languages/en.yml#L18

Reply:
@bahmutov it's used for format string, for example here:
https://github.com/cypress-io/cypress-documentation/blob/b8a035ca4b4d1e9deaa7721216865330d7cccd96/themes/cypress/layout/api-toc.swig#L22

Hexo documentation says that "There is support for the printf format in language files": https://hexo.io/docs/internationalization#Language-Files

So it allows to convert date of the last update to string and use in the template :)

Reply:
interesting, I did not notice that this could be customized per language